resume.yaml

```diff
@@ -1,121 +0,0 @@
-name: Sean Gatewood
-contact:
-  email: stg2bd@virginia.edu
-  linkedin: linkedin.com/in/sean-t-gatewood
-  github: github.com/sgatewood
```

Comment:
I’m partial to icons and handles because I think it’s a bit more modern than URLs, but up to you

Reply:
I had to base64 encode them 🥲